examples/mqtt_client/mqtt_client.c

```diff
@@ -0,0 +1,249 @@
+
+#include "espressif/esp_common.h"
+#include "esp/uart.h"
+
+#include <string.h>
+
+#include <FreeRTOS.h>
+#include <task.h>
+#include <ssid_config.h>
+
+#include <espressif/esp_sta.h>
+#include <espressif/esp_wifi.h>
+
+#include <paho/MQTTESP8266.h>
+#include <paho/MQTTClient.h>
+
+#include <semphr.h>
+
```

Comment:
Can I just put LICENSE.txt file from BAOSHI's project into extras/paho/ or better add contents of LICENSE.txt as header in each file?